src/mame/apple/apple2e.cpp

```diff
@@ -2237,11 +2149,13 @@ void apple2e_state::c000_w(offs_t offset, u8 data)
 	{
 		case 0x00:  // 80STOREOFF
 			m_80store = false;
+			m_video->m_80store = false;
 			auxbank_update();
 			break;
 
 		case 0x01:  // 80STOREON
 			m_80store = true;
+			m_video->m_80store = true;
 			auxbank_update();
 			break;
```

Comment:
Is there a better way of dealing with this than having a public data member in the device that mirrors the a driver state member?
* Duplicate state is usually a red flag, and runs the risk of getting out-of-sync.
* If they aren’t actually the same flag, the identical names could cause confusion.
* Public data members, particularly when they’re set from outside the class, are often a red flag themselves.

Reply:
That's my bad, he's just following along.  The A2 video isn't easy to draw a firm demarcation around, but those flags should belong to the video system and be set/queried by the driver as appropriate rather than stored in duplicate.

Reply:
I agree with the criticisms, but it's not obvious to me how to fix it properly, and I think it should be deferred to a later commit.

Reply:
I'll handle that aspect of it since it's my problem in the first place and I know how I want it cleaned up.